README.md

```diff
@@ -94,6 +95,22 @@ It is best practice to build the database, make changes to `database.sqlite`, an
 
 **Docker** - `docker-compose up build-json`
 
+## Import
```

Comment:
Line 78 above needs to incorporate this import process or be made generic to apply to all 3 sub-headings.

Reply:
Done